Huion_Kamvas_GT191/descriptors.txt

```diff
@@ -11,8 +11,8 @@ Device Descriptor:
   idVendor           0x256c 
   idProduct          0x006e 
   bcdDevice            0.00
-  iManufacturer           5 (error)
-  iProduct                6 (error)
+  iManufacturer           5 Huion
+  iProduct                6 Kamvas GT191
```

Comment:
Hmm, I'm not sure we should overwrite this. Have you got an lsusb output for one of these tablets which has these strings?